src/cgame/rocket/rocket_documents.cpp

```diff
@@ -60,10 +60,15 @@ void Rocket_LoadDocument( const char *path )
 // Scale the UI proportional to the screen size
 void Rocket_SetDocumentScale( Rml::Core::ElementDocument& document )
 {
-	// This gets 12pt on 1920�1080 screen, which is libRocket default for 1em
-	int size = std::min( cgs.glconfig.vidWidth, cgs.glconfig.vidHeight ) / 90;
+	// This sets the default font size to 18px on a 1920�1080 screen,
+	// and scales appropriately.
+	int size = std::min( cgs.glconfig.vidWidth, cgs.glconfig.vidHeight ) / 60;
+	// Note that this enforces the font-size on the root element. So any
+	// value you would set on that same element (<window> or <body>) would
+	// be forcibly overriden by this.
+	// Use a rule such as `body > * { font-size: 1.2em; }` to adjust size.
 	document.SetProperty( Rml::Core::PropertyId::FontSize,
```

Comment:
We're using `em` units everywhere, so how does this not throw off everything in the entire UI?

Reply:
This PR is to be tested with https://github.com/UnvanquishedAssets/unvanquished_src.dpkdir/pull/113. I made sure that thing look better than they currently do.

To answer your question, « We're using em units everywhere, so how does this not throw off everything in the entire UI? », well we had that problem already when switching to RmlUI: it changed the font size, and we can see it broke a fair bit of things. This time the font size changes are made on purpose, at least :smiley:

Reply:
I think there is a better way in RmlUi that would let us set a font size on a root element without this wacky `body > *` stuff. Instead of setting the font size here we can use the `SetDensityIndependentPixelSize` function on a context. Then the font size in the CSS can be set using the [dp](https://mikke89.github.io/RmlUiDoc/pages/rcss/syntax#dp-unit) unit.